entgql/schema.go

```diff
@@ -361,13 +361,7 @@ func (e *schemaGenerator) buildDirectives(directives []Directive) ast.DirectiveL
 	for _, d := range directives {
 		args := make(ast.ArgumentList, 0, len(d.Arguments))
 		for _, a := range d.Arguments {
```

Comment:
This loop is not need anymore. Just assign from the d.Argument

Reply:
I totally missed that one, thank you for pointing it out. It's refactored 👍🏻 .